Release influxctl v2.9.5 and v2.9.6 (#5565)
influxdata/docs-v2: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md

---
title: influxctl auth
description: >
  The `influxctl auth` command and its subcommands provide the ability to
  login and logout.
menu:
  influxdb_cloud_dedicated:
    parent: influxctl
weight: 201
---

The `influxctl auth` command and its subcommands provide information about
{{< product-name >}} clusters.

## Usage

```sh
influxctl auth [subcommand] [subcommand options] [arguments...]
```

## Subcommands

| Subcommand                                                               | Description                     |
| :----------------------------------------------------------------------- | :------------------------------ |

> | [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Log in using InfluxData Auth0   |

Reviewer: ```suggestion
| [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Log in to an InfluxDB cluster using InfluxData Auth0 |
```

> | [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Login with InfluxData Auth0 or InfluxDB Clustered identity provider |

Reviewer: ```suggestion
| [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Log in using InfluxData Auth0 or an InfluxDB Clustered identity provider |
```

> | [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Login with InfluxData Auth0 or InfluxDB Clustered identity provider |

Reviewer: > InfluxDB Clustered identity provider

Does this only apply to Clustered, or to Cloud Dedicated as well?

Author: Both, dedicated is the InfluxData Auth0

Author: Right

> | [logout](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout/) | Remove local tokens             |

Reviewer: ```suggestion
| [logout](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout/) | Remove local authorization tokens to log out of an InfluxDB cluster            |
```

> | [logout](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout/) | Remove local authorization tokens to log out of an InfluxDB cluster            |

Reviewer: ```suggestion
| [logout](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout/) | Log out of an InfluxDB cluster; remove local authorization tokens |
```
